### PR TITLE
create model person

### PR DESCRIPTION
**Pull request by acmesquita**
Create model person

### CODE REVIEW

File: db/migrate/20170903200617_create_person.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreatePerson < ActiveRecord::Migration[5.1]
+  def change
+    create_table :people do |t|
+
+      t.string :name
+      t.date :bith
```
Comment (marclerodrigues) [inline db/migrate/20170903200617_create_person.rb]: I believe here the correct is `birthday`

**Comment by marclerodrigues** [comment]
After you correct what I've mentioned you can merge this pull request.

File: db/migrate/20170903200617_create_person.rb (hunk 1 above)
Comment (acmesquita) [reply db/migrate/20170903200617_create_person.rb]: ok..I gonna change

**Comment by acmesquita** [comment]
create model Person

**Pull request merged**